File: wallendar-app/app/src/main/java/com/td/wallendar/dtos/response/UserAliasResponse.java
package com.td.wallendar.dtos.response;

import com.td.wallendar.models.UserAlias;

public class UserAliasResponse {
    private long id;
    private String alias;

    public UserAliasResponse() {
    }

    public UserAliasResponse(UserAlias userAlias) {
        this.id = userAlias.getId();
        this.alias = userAlias.getAlias();
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getAlias() {
        return alias;
    }

    public void setAlias(String alias) {
        this.alias = alias;
    }
}

File: wallendar-app/app/src/main/java/com/td/wallendar/dtos/response/UserAliasResponse.kt
package com.td.wallendar.dtos.response

import com.td.wallendar.models.UserAlias

class UserAliasResponse(userAlias: UserAlias) {
    private var id: Long = 0
    private var alias: String

    init {
        id = userAlias.id
        alias = userAlias.alias
    }

    fun getId(): Long {
        return id
    }

    fun setId(id: Long) {
        this.id = id
    }

    fun getAlias(): String{
        return alias
    }

    fun setAlias(alias: String) {
        this.alias = alias
    }
}